src/components/EditCoursePage/AdditionalMetadataFields.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+import RenderInputTextField from '../RenderInputTextField';
+import FieldLabel from '../FieldLabel';
+import RichEditor from '../RichEditor';
+
+function AdditionalMetadataFields(props) {
+  const {
+    disabled,
+  } = props;
+
+  return (
+    <div className="collapsible-card pgn_collapsible mt-4">
+      <div className="collapsible-body">
+        <FieldLabel text="Executive Education details" className="h3 font-weight-normal mb-3" />
+        <Field
+          name="additional_metadata.external_url"
+          component={RenderInputTextField}
+          label=<FieldLabel id="external_url.label" text="External landing page URL" />
```

Comment:
Please use the Sentence case for the labels. I forgot to mention this earlier.

Reply:
I've followed the same we had in our mockups. Should I update this?

Reply:
Yes, please update this.

Reply:
Apologies, I confused sentence case with title case. It is correct as it is. Ignore my comment above.